scripts/custom_objects_spec.json

```diff
@@ -0,0 +1,554 @@
+{
+  "/apis/{group}/{version}/{kind}": {
+    "parameters": [
+      {
+        "uniqueItems": true,
+        "type": "string",
+        "description": "If 'true', then the output is pretty printed.",
+        "name": "pretty",
+        "in": "query"
+      },
+      {
+        "name": "group",
+        "in": "path",
+        "required": true,
+        "description": "The Custom object's group name",
+        "type": "string"
+      },
+      {
+        "name": "version",
+        "in": "path",
+        "required": true,
+        "description": "The Custom object's version",
+        "type": "string"
+      },
+      {
+        "name": "kind",
```

Comment:
Yes, this is the one goes into URL. Do you have any suggestion for its name that make sense in both CRD and TPR world?